src/background/background-backup-wallet-salt-handler.ts

```diff
@@ -0,0 +1,88 @@
+import { logger } from '@shared/logger';
```

Comment:
Curious about this file now as separate from `backup-old-wallet-salt.ts`, do they need to be separate or named in a way that they can be one file? I def don't think `background` needs to be in the file name here?

Reply:
Agree, I will club them together then and rename. Yes background should be removed.